fixtures/features/src/index.ts

```diff
@@ -5,10 +5,3 @@ function render() {
 }
 
 render();
-
-// Uncomment to enable HMR with Vite
-// if (import.meta.hot) {
-//   import.meta.hot.accept('./features.css', () => {
-//     render();
-//   });
-// }
```

Comment:
This doesn't belong here anymore since this file no longer imports `features.css.ts`.